analyzer/tools/build-logger/src/ldlogger-tool-gcc.c

```diff
@@ -343,6 +343,40 @@ int isObjectFile(const char* filename_)
   return 0;
 }
 
+/**
+ * Reads the arguments from a response file.
+ *
+ * @param file_ a file that contains the arguments.
+ * @param args_ a vector for arguments.
+ */
+static void readArgumentsFromFile(const char* file_, LoggerVector* args_)
+{
+  const char* delimiters = " \n\"";
```

Comment:
Considering " character as a delimiter is suspicious for me. Let's keep in mind that ' character may also play the same role as ". Moreover, there are some test cases for the logger that check the usage of quoted arguments. Could you check these too?
https://github.com/Ericsson/codechecker/blob/3e8bc722ac7d61af6850999e92dd3fb612092a62/analyzer/tools/build-logger/test/test_logger.sh#L87-L109